files/mods.json

```diff
@@ -760,7 +759,7 @@
         "creator": "DankerWanker",
         "discordcode": "BUmywXX47M",
         "command": "/dsm",
-        "description": "DSM provides dungeon puzzle solvers, a reparty command, custom boss music and more useful features to see for yourself.",
+        "description": "Dungeon puzzle solvers, a reparty command, custom boss music and more useful features to see for yourself",
```

Comment:
other mods do the solvers better and have a reparty command. might be better to replace these with something only dsm has, like some of their specific trackers

Reply:
could you provide suggested changes

Reply:
fishing counters+stats, lock slayers, maddox dial (main big sbe features that don't have other mods for alternatives, or at least big in my eyes I don't play sb)